README.md

```diff
@@ -200,6 +201,14 @@ The following CMake options are available for adding/removing project features.
 
       i.e. cmake -DTRANSPORT_CURL=ON ..
 
+### VSCode
+
+For convenience, you can quickly get started using [VSCode](https://code.visualstudio.com/) and the [CMake Extension by Microsoft](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools&ssr=false#overview). Included in the repo is a `settings.json` file [here](https://github.com/Azure/azure-sdk-for-c/blob/master/.vscode/settings.json) which the extension will use to configure a CMake project. With this, you can run and debug samples and tests. Modify the variables in the file to your liking or as instructed by sample documentation and then select the following button in the extension:
+
+![VSCode CMake Config](./sdk/docs/resources/vscode_cmake_config.png)

```

Comment:
Why did we choose `sdk/docs/resources/` for this image, when the README is a the root?

Should we move it to `<root of repo>/resources`?